cmd/caa-log-checker/main.go

```diff
@@ -193,23 +194,21 @@ func removeCoveredTimestamps(timestamps []time.Time, cover time.Time, tolerance
 	return r
 }
 
-// formatErrors returns nil if the input map is empty. Otherwise, it returns an
-// error containing a listing of every name and issuance time that was not
-// covered by a CAA check.
-func formatErrors(remaining map[string][]time.Time) string {
+// formatErrors returns nil if the input map is empty. Otherwise, it logs
+// a line for each name and issuance time that was not covered by a CAA
+// check, and return an error.
+func formatErrors(log blog.Logger, remaining map[string][]time.Time) error {
```

Comment:
I'd rename from `formatErrors` to `emitErrors` or something like that, now that it's actually doing logging instead of just producing the error string.